react-big-calendar/index.d.ts

```diff
@@ -26,10 +26,10 @@ declare module 'react-big-calendar' {
         step?: number;
         rtl?: boolean;
         eventPropGetter?: (event: Object, start: stringOrDate, end: stringOrDate, isSelected: boolean) => void;
-        titleAccessor?: string;
-        allDayAccessor?: boolean;
-        startAccessor?: stringOrDate;
-        endAccessor?: stringOrDate;
+        titleAccessor?: string | ((row: Object) => string);
+        allDayAccessor?: string | ((row: Object) => boolean);
```

Comment:
I hear ya.  I was trying not to get too invested in this type declaration, but perhaps a step in a better direction still would be to use type inference:

interface BigCalendarProps**<T>** extends React.Props<BigCalendar>

events?: **T**[];

endAccessor?: **keyof T** | ((row: **T**) => Date);